kmir/k-src/mir-assert.md

```diff
@@ -0,0 +1,38 @@
+```k
+require "mir-operand.md"
+```
+
+```k
+module MIR-ASSERT-SYNTAX 
+  import MIR-OPERAND-SYNTAX
+
+  syntax AssertKind ::= BoundsCheck
+                      | Overflow
+                      | OverflowNeg
+                      | DivisionByZero
+                      | RemainderByZero
+                      | ResumedAfterReturn
+                      | ResumedAfterPanic
+                      | MisalignedPointerDereference
+
+  syntax BoundsCheck ::= "\"index out of bounds: the length is {} but the index is {}\"" "," Operand "," Operand //Should this Oprand always uSize
+  syntax OverflowNeg ::= "\"attempt to negate " Operand ", which would overflow\""
+  syntax DivisionByZero ::= "\"attempt to divide `{}` by zero\"" "," Operand
+  syntax RemainderByZero ::= "\"attempt to calculate the remainder of " Operand " with a divisor of zero\""
```

Comment:
Here and also below - it probably does not matter, but you're using spaces at the end of some tokens. Is it because you want to make sure that the last word ends at that point?

Reply:
No I think that is just a mistake. I removed it